### DISCUSSION TITLE
Q3 of Day 3

### DISCUSSION THREAD

**Discussion by usadev0813** [I-JUNNYLAND-I/algorithm#38] [Q&A] [answered]
s의 패턴이 (), {}, []와 같이 정해져 있는 줄 알았습니다. (  "{()}" 와 같은 경우를  생각 못했음 )

**Comment by Junnyjun** [comment, marked as answer]
({[ 순서로 열릴때는 쌓아놓고 ]닫힐때는 하나씩 상쇄시키면 어떨까요?

ex ) ({[ ]})
1. ({[ (쌓음)
2. ] 지움
3. } 지움
4. ) 지움
다지워지지 않앗으면 false
